Test_Automation/README

```diff
@@ -0,0 +1,75 @@
+# * Copyright (c) 2020-2021, Happiest Minds Technologies Limited Intellectual Property. All rights reserved.
+# *
+# * SPDX-License-Identifier: LGPL-2.1-only
+
+# This section is divided in to 3 parts:
```

Comment:
This file would benefit from becoming a README.md using Markdown https://guides.github.com/features/mastering-markdown/

Reply:
Agree on this Nick. I will update it in that format. Looks like need to create README.md file and remove the existing one

Reply:
Hi Nick, added new README.md file and deleted the old one